csrc/executor.cpp

```diff
@@ -101,7 +101,7 @@ std::string getStructuredCodeFromExternalFiles(const int64_t fusion_id) {
     return "";
   }
   std::string all_external_code_paths(external_code_path);
-  if (all_external_code_paths.empty() || fusion_id < 1) {
+  if (all_external_code_paths.empty() || fusion_id < 0) {
```

Comment:
@rdspring1 This check on `fusion_id` could possibly be removed. I changed it here to check that `fusion_id != -1`.